sandy/core/endf6.py

```diff
@@ -1408,7 +1408,8 @@ def merge_pendf(self, pendf):
                    .add_sections(section_3_pendf) \
                    .add_sections(section_1451_pendf)
 
-    def get_errorr(self, njoy, pendf=None):
+    def get_errorr(self, njoy, pendf=None, temperatures=0, suffixes=0,
+                   group_structure=None):
```

Comment:
or maybe `group_structure` can be 2 by default, which corresponds to the IWT number. And in case an iterable is given, the code would directly switch to ITW=1 and use `ek`